esx_service/utils/vmdk_utils.py

```diff
@@ -42,18 +42,21 @@
 SNAP_SUFFIX_GLOB = "-[0-9][0-9][0-9][0-9][0-9][0-9].vmdk"
 
 
-def get_datastores():
+def get_datastores(doUpdate = False):
```

Comment:
Isn't it better to always update the list in case a datastore isn't found vs. one more function to decide on that.